src/self-help/azext_self_help/aaz/latest/self_help/solution/_create.py

```diff
@@ -0,0 +1,922 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "self-help solution create",
+    is_preview=True,
+)
+class Create(AAZCommand):
+    """Create a solution for the specific Azure resource or subscription using the triggering criteria from discovery solutions.
+
+    :example: Create Solution at Resource Level
+        az self-help solution create --solution-name {solution-name} --trigger-criteria {trigger-criteria} --parameters {parameters} --scope {scope}
```

Comment:
Recommend using fake values instead of placeholder in examples.
```suggestion
        az self-help solution create --solution-name solution-name --trigger-criteria trigger-criteria --parameters parameters --scope scope
```